app-misc/wcd/wcd-6.0.4.ebuild

```diff
@@ -0,0 +1,51 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit toolchain-funcs
+
+DESCRIPTION="Wherever Change Directory"
+HOMEPAGE="http://waterlan.home.xs4all.nl/#WCD_ANCHOR"
+SRC_URI="http://waterlan.home.xs4all.nl/${PN}/${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~x86 ~amd64-linux ~x86-linux"
+IUSE="nls unicode"
+
+CDEPEND="
```

Comment:
Just change this to DEPEND and then define RDEPEND below, i.e. no need for a separate CDEPEND var